lib/packages/core/src/api/__tests__/transactionApi.spec.ts

```diff
@@ -70,7 +71,7 @@ describe('Transaction Api', () => {
             httpMock = HttpMockBuilder.create()
             // tslint:disable:max-line-length
             .onPostReply(200, mockBroadcastResponse,
-                'relPath?requestType=sendMoney&requestType=sendMoney&amountNQT=100000000&publicKey=recipientId&recipient=senderPrivateKey&deadline=1440&feeNQT=100000000')
```

Comment:
wow, how did this passed PR?

Reply:
yeah, my fault... :/ negative testing would've caught this.

Reply:
Although it's caused becasue I was passing the arguments in the wrong order. It wasn't a problem in the function itself, just the test 👍